frontend/components/text-classifier/results/ClassifierAnnotationArea.vue

```diff
@@ -122,7 +156,7 @@ export default {
     },
     filteredLabels() {
       return this.sortedLabels.filter((label) =>
-        label.class.toLowerCase().match(this.searchText)
+        label.class.toLowerCase().match(this.idState ? this.searchText : undefined)
```

Comment:
Why `this.idState` could  `undefined` ?

Reply:
I think the watcher interfered with idState

Reply:
Then we can move `this.idState` validation inside `searchText` getter

```javascript
searchText: {
      get: function () {
        return this.idState ? this.idState.searchText : undefined;
      },
```
And, what about the rest of computed fields ? Should we need validate `idState` for all of them since it could be `undefined`?